docs/src/app/components/CodeExample/index.jsx

```diff
@@ -3,20 +3,7 @@ import CodeBlock from './CodeBlock';
 import ClearFix from 'material-ui/lib/clearfix';
 import Paper from 'material-ui/lib/paper';
 
-const CodeExample = React.createClass({
-
-  propTypes: {
```

Comment:
It would be better! I guess I can use https://github.com/zertosh/class-props-codemod.